README.md

```diff
@@ -128,6 +128,30 @@ if (!supportsTime) {
 
 This will add a global `TimePolyfill` function to the page.
 
+### Shadow DOM
+
+When your code is inside a component that resides in the Shadow DOM, the polyfill may not be able to find your label
+element. For this case, you can pass your label element in directly.
+
+```
+<label id="myLabel"></label>
+<input #time type="time" id="timeInput">
+```         
+
+```               
+// This example is in Angular/Typscript
+
+import * as timePolyFill from 'time-input-polyfill';
+
+public constructor(private elemRef: ElementRef) {
```

Comment:
I want to display the simplest syntax possible.

The example shouldn't use class syntax.

Try to make the example as simple as possible.